src/DIRAC/FrameworkSystem/scripts/dirac_login.py

```diff
@@ -304,7 +314,27 @@ def loginWithCertificate(self):
             # Upload proxy to the server if it longer that uploaded one
             if credentials["secondsLeft"] > uploadedProxyLifetime:
                 gLogger.notice("Upload proxy to server.")
-                return gProxyManager.uploadProxy(proxy)
+                res = gProxyManager.uploadProxy(proxy)
+                if not res["OK"]:
+                    return res
```

Comment:
```suggestion
                if not (res := gProxyManager.uploadProxy(proxy))["OK"]:
                    return res
```

Reply:
For the time being, I despise the walrus operator so I won't be writing that :-D